src/engine/effects/engineeffectchain.cpp

```diff
@@ -101,25 +111,28 @@ bool EngineEffectChain::processEffectsRequest(const EffectsRequest& message,
             break;
         case EffectsRequest::SET_EFFECT_CHAIN_PARAMETERS:
             if (kEffectDebugOutput) {
-                qDebug() << debugString() << "SET_EFFECT_CHAIN_PARAMETERS"
+                qDebug() << debugString() << this << "SET_EFFECT_CHAIN_PARAMETERS"
                          << "enabled" << message.SetEffectChainParameters.enabled
                          << "mix" << message.SetEffectChainParameters.mix;
             }
             response.success = updateParameters(message);
             break;
-        case EffectsRequest::ENABLE_EFFECT_CHAIN_FOR_CHANNEL:
+        case EffectsRequest::ENABLE_EFFECT_CHAIN_FOR_INPUT_CHANNEL:
             if (kEffectDebugOutput) {
-                qDebug() << debugString() << "ENABLE_EFFECT_CHAIN_FOR_CHANNEL"
+                qDebug() << debugString() << this << "ENABLE_EFFECT_CHAIN_FOR_CHANNEL"
+                         << message.pTargetChain
                          << message.channel;
             }
-            response.success = enableForChannel(message.channel);
+            response.success = enableForInputChannel(message.channel,
+                    message.pStatesForEffectsInChain);
             break;
-        case EffectsRequest::DISABLE_EFFECT_CHAIN_FOR_CHANNEL:
+        case EffectsRequest::DISABLE_EFFECT_CHAIN_FOR_INPUT_CHANNEL:
             if (kEffectDebugOutput) {
-                qDebug() << debugString() << "DISABLE_EFFECT_CHAIN_FOR_CHANNEL"
+                qDebug() << debugString() << this << "DISABLE_EFFECT_CHAIN_FOR_CHANNEL"
```

Comment:
remove debug

Reply:
kEffectDebugOutput is set to false in message.h